src/solvers/nonlinear_solid_solver.cpp

```diff
@@ -111,7 +111,7 @@ void NonlinearSolidSolver::setSolverParameters(const serac::LinearSolverParamete
 void NonlinearSolidSolver::completeSetup()
 {
   // Define the nonlinear form
-  H_form_ = std::make_shared<mfem::ParNonlinearForm>(displacement_->space.get());
+  auto H_form_ = std::make_unique<mfem::ParNonlinearForm>(displacement_->space.get());
```

Comment:
Since `H_form_`, `M_form_`, and `S_form_` aren't members anymore, we should drop the trailing underscore.

Reply:
Good catch, I had left them that way to just test if they could work as locals.